src/crepe/facade/SDLContext.h

```diff
@@ -4,6 +4,7 @@
 #include <SDL2/SDL_keycode.h>
 #include <SDL2/SDL_rect.h>
 #include <SDL2/SDL_render.h>
+#include <SDL2/SDL_ttf.h>
```

Comment:
Het is een facade header dus het maakt niet veel uit maar deze header wordt niet gebruikt in dit .h bestand en zou alleen in SDLContext.cpp hoeven te staan toch?